app/services/catalog/submit_order.rb

```diff
@@ -43,21 +47,6 @@ def parameters(item)
       end
     end
 
-    def update_item(item, result)
-      item.topology_task_ref     = result.task_id
-      item.state                 = 'Ordered'
-      item.order_request_sent_at = Time.now.utc
```

Comment:
I think you missed this attribute. You could either pass it in up on line 38 or probably include it by default in the `mark_ordered` method in the `OrderItem` class itself.

Reply:
Adding it to the `mark_ordered` method, similar to how I'm doing the `:completed_at` in the mark failed/completed methods.